preseed/provision.erb

```diff
@@ -146,4 +152,4 @@ d-i grub-installer/bootdev string default
 <% end -%>
 d-i finish-install/reboot_in_progress note
 
-d-i preseed/late_command string wget -Y off <%= @static ? "'#{foreman_url('finish')}&static=true'" : foreman_url('finish') %> -O /target/tmp/finish.sh && in-target chmod +x /tmp/finish.sh && in-target /tmp/finish.sh
+d-i preseed/late_command string wget -Y off <%= dhcp ? foreman_url('finish') : "'#{foreman_url('finish')}&static=true'" %> -O /target/tmp/finish.sh && in-target chmod +x /tmp/finish.sh && in-target /tmp/finish.sh
```

Comment:
Perhaps keep the conditional based on `@static` here, so we don't begin passing static=yes to finish scripts when a subnet is set to static.  Finish scripts can still check for the subnet boot mode.